ArduPlane/Attitude.cpp

```diff
@@ -670,23 +670,40 @@ void Plane::update_load_factor(void)
     float max_load_factor =
         sq(smoothed_airspeed / MAX(stall_airspeed_1g, 1));
 
+    // allow limiting roll command down to wings-level when necessary if
+    // airspeed is accurate and airspeed stall is set (implying low load factor
+    // overhead)
+    bool enforce_full_roll_limit =
```

Comment:
```suggestion
    const bool enforce_full_roll_limit =
```

Reply:
`const` anything everywhere, all at once.

We might not use best practises everywhere, but we can move towards that goal :-)